### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@peternewman any idea why the CI fails here? I don't understand where this issues is manifesting from: `error consistes: multiple corrections must end with trailing ","`

**Comment by peternewman** [comment]
> @peternewman any idea why the CI fails here? I don't understand where this issues is manifesting from: `error consistes: multiple corrections must end with trailing ","`

Looks like you've cracked it!

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -33566,6 +33624,9 @@ vesions->versions
 vetex->vertex
 vetexes->vertices
 vetod->vetoed
+vetor->vector
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:33627]:
```suggestion
vetor->vector, veto,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -33566,6 +33624,9 @@ vesions->versions
...
+vetored->vectored
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:33628]:
```suggestion
vetored->vectored, vetoed,
```

**Comment by luzpaz** [comment]
The error doesn't indicate the line number it's on

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -33566,6 +33624,9 @@ vesions->versions
...
+vetors->vectors
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:33629]: Maybe:
```suggestion
vetors->vectors, vetoes,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -31895,10 +31947,15 @@ traslucency->translucency
...
 trasnaction->transaction
+trasnform->transform
+trasnformation->transformation
+trasnformed->transformed
+trasnforms->transforms
 trasnlate->translate
 trasnlated->translated
 trasnlation->translation
 trasnlations->translations
+trasnparent->transparent
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:31958]:
```suggestion
trasnparencies->transparencies
trasnparency->transparency
trasnparent->transparent
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -31895,10 +31947,15 @@ traslucency->translucency
 trasmission->transmission
 trasmit->transmit
 trasnaction->transaction
+trasnform->transform
+trasnformation->transformation
+trasnformed->transformed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:31952]:
```suggestion
trasnformed->transformed
trasnformer->transformer
trasnformers->transformers
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -26530,6 +26575,10 @@ reprensent->represent
 reprensentation->representation
 reprensentational->representational
 reprensentations->representations
+reprent->represent, reprint,
+reprented->represented, reprinted,
+reprenting->representing
+reprents->represents, reprints,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26581]:
```suggestion
reprents->represents, reprints, repents,
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -26530,6 +26575,10 @@ reprensent->represent
 reprensentation->representation
 reprensentational->representational
 reprensentations->representations
+reprent->represent, reprint,
+reprented->represented, reprinted,
+reprenting->representing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26580]:
```suggestion
reprent->represent, reprint, repent,
reprented->represented, reprinted, repented,
reprenting->representing, repenting,
```

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -24760,6 +24803,7 @@ queisce->quiesce
 queriable->queryable
 quering->querying
 querries->queries
+querry->query
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24806]:
```suggestion
querry->query, quarry,
```

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -17124,6 +17148,7 @@ indefintly->indefinitely
 indempotent->idempotent
 indendation->indentation
 indended->intended, indented,
+indendent->independent, indented,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:17151]: Maybe:
```suggestion
indendent->independent, indented, indent,
```

**Review by peternewman — CHANGES_REQUESTED** [review]

**Pull request merged**